about/release_policy.rst

```diff
@@ -134,21 +132,32 @@ provided for testing purposes only.
 
 .. seealso::
 
-    See :ref:`doc_upgrading_to_godot_4` for instructions on migrating a project.
+    See :ref:`doc_upgrading_to_godot_4` for instructions on migrating a project
+    from Godot 3.x to 4.x.
 
 .. _doc_release_policy_when_is_next_release_out:
 
 When is the next release out?
 -----------------------------
 
 While Godot contributors aren't working under any deadlines, we strive to
-publish minor releases relatively frequently, with an average of two 3.x minor
-releases per year since Godot 3.3.
+publish minor releases relatively frequently.
+
+In particular, after the very length release cycle for 4.0, we are pivoting to
+a faster paced development workflow, with the 4.1 release expected within late
+Q2 / early Q3 2023.
+
+Frequent minor releases will enable us to ship new features faster (possibly
+as experimental), get user feedback quickly, and iterate to improve those
+features and their usability. Likewise, the general user experience will be
+improved more steadily with a faster path to the end users.
 
 Maintenance (patch) releases are released as needed with potentially very
 short development cycles, to provide users of the current stable branch with
 the latest bug fixes for their production needs.
 
-As for the upcoming Godot 4.0, as of December 2022, we are well into the *beta*
-phase, and are aiming for a stable release in Q1 2023.
-`Follow the Godot blog <https://godotengine.org/news>`__ for the latest updates.
+The 3.6 release is still planned and should be the last stable branch of Godot
+3.x. It will be a Long Term Support (LTS) release, which we plan to support for
```

Comment:
```suggestion
3.x. It will be a Long-Term Support (LTS) release, which we plan to support for
```

I wouldn't capitalize it either, but up to you.